doc/index.rst

```diff
@@ -68,6 +72,9 @@ For ``OFFLINE`` state, run the script this way.
 
   shutdown_dsm.py -f
 
+In both cases, the script defaults to index=1.
+You must ensure that the index used by the shutdown script is commensurate with the one used by the run script.
```

Comment:
Please consider "... is commensurate with ..." -> "... matches ..." or perhaps "Please be sure to specify the correct index for the DSM CSC you wish to terminate".

Reply:
This has now changed with the adjustments made above.